admin/package-files/docker-plugin/upstart/flocker-docker-plugin.conf

```diff
@@ -0,0 +1,11 @@
+# flocker-docker-plugin - flocker-docker-plugin job file
+
+description "Flocker Docker Plugin"
+author "ClusterHQ <support@clusterhq.com>"
+
+start on starting docker
+stop on stopped docker
+respawn
+
+# Start the process
+exec /usr/sbin/flocker-docker-plugin --logfile=/var/log/flocker/flocker-docker-plugin.log
```

Comment:
I had to transfer the `user.{crt,key}` from the acceptance test certificate directory on my laptop to `/etc/flocker/api.{crt,key}` on the node.

After that the plugin starts up
```
root@acceptance-test-richardw-0:~# tail /var/log/flocker/flocker-docker-plugin.log 
...
{"task_uuid": "b87ee149-7e2d-4375-97fd-3626318ab013", "error": false, "timestamp": 1439890573.14672, "message": "Log opened.", "message_type": "twisted:log", "task_level": [1]}
{"task_uuid": "89631396-70f9-46a0-9b0b-504abe835cee", "error": false, "timestamp": 1439890573.195605, "message": "Site starting on '/run/docker/plugins/flocker/flocker.sock'", "message_type": "twisted:log", "task_level": [1]}
{"task_uuid": "56c5ce16-3c96-42d9-9e28-437000a0497d", "error": false, "timestamp": 1439890573.195946, "message": "Starting factory <twisted.web.server.Site instance at 0x7f44e476eb00>", "message_type": "twisted:log", "task_level": [1]}

```


```
root@acceptance-test-richardw-0:~# ls -l /run/docker/plugins/flocker/flocker.sock
srw------- 1 root root 0 Aug 18 09:36 /run/docker/plugins/flocker/flocker.sock
```